`npm run start` can fail if typescript build files are present

<!-- Hello! Please read the [Contributing Guidelines](CONTRIBUTING.md) before submitting an issue. -->

## Description of the issue

```
❯ run start

> start
> wireit

                                                     
> @patternfly/pfe-core@2.4.1 analyze
> cem analyze --watch

[3:42:08 PM] @custom-elements-manifest/analyzer: Created new manifest.
                                                     
❌ [build:elements] exited with exit code 2. Output:

error TS5055: Cannot write file '/home/bennyp/Developer/patternfly/patternfly-elements/elements/pf-chip/pf-chip-shared.css.js' because it would overwrite input file.


Found 1 error.

file:///home/bennyp/Developer/patternfly/patternfly-elements/node_modules/wireit/lib/execution/service.js:15
    return new Error(`Unexpected service state ${state.id}`);
           ^

Error: Unexpected service state executingDeps
    at unexpectedState (file:///home/bennyp/Developer/patternfly/patternfly-elements/node_modules/wireit/lib/execution/service.js:15:12)
    at ServiceScriptExecution.start (file:///home/bennyp/Developer/patternfly/patternfly-elements/node_modules/wireit/lib/execution/service.js:575:23)
    at Executor.execute (file:///home/bennyp/Developer/patternfly/patternfly-elements/node_modules/wireit/lib/executor.js:133:42)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (file:///home/bennyp/Developer/patternfly/patternfly-elements/node_modules/wireit/lib/cli.js:123:52)
    at async file:///home/bennyp/Developer/patternfly/patternfly-elements/node_modules/wireit/lib/cli.js:156:16

Node.js v20.10.0```

### Expected behavior

run start should always work

--- a/.changeset/fix-start-build-artifacts.md
+++ b/.changeset/fix-start-build-artifacts.md
@@ -1,0 +1,6 @@
+---
+"@patternfly/elements": patch
+---
+
+`npm run start` no longer fails when TypeScript build artifacts
+are present in the working tree.

--- a/tsconfig.esbuild.json
+++ b/tsconfig.esbuild.json
@@ -9,7 +9,16 @@
     "**/demo/*.d.ts",
     "**/_temp/**/*",
     "**/*.story.*",
-    "**/test/*_e2e*"
+    "**/test/*_e2e*",
+    "elements/**/*.js",
+    "elements/**/*.d.ts",
+    "elements/**/*.js.map",
+    "core/**/*.js",
+    "core/**/*.d.ts",
+    "core/**/*.js.map",
+    "tools/**/*.js",
+    "tools/**/*.d.ts",
+    "tools/**/*.js.map"
   ],
   "compilerOptions": {
     "allowJs": true,

